templates/index.html

```diff
@@ -31,11 +31,179 @@
+            <td></td>
+            <td><a href="https://mir-server.io/docs">Mir</a></td>
+            <td><a href="https://canonical-netplan.readthedocs-hosted.com/en/latest/">Netplan</a></td>
+            <td></td>
+            <td><a href="https://documentation.ubuntu.com/charmed-mlflow/en/latest/">Charmed MLflow</a></td>
+            <td><a href="https://canonical.com/data/docs">Charmed Data</a></td>
+          </tr>
+          <tr style="background: rgba(0,0,0,0.05); border-top: 1px solid rgba(0,0,0,0.05);">
+            <th style="background: rgba(255, 255, 255, 1); border-top: 1px solid rgba(255,255,255,1)"><strong></strong></th>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td><a href="https://canonical-microceph.readthedocs-hosted.com/en/reef-stable/">MicroCeph</a></td>
+            <td><a href="https://docs.cloud-init.io/en/latest/">cloud-init</a></td>
+          </tr>
+          <tr style="background: rgba(0,0,0,0.05); border-top: 1px solid rgba(0,0,0,0.05);">
+            <th style="background: rgba(255, 255, 255, 1); border-top: 1px solid rgba(255,255,255,1)"><strong></strong></th>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td><a href="https://canonical-microovn.readthedocs-hosted.com/en/latest/">MicroOVN</a></td>
+            <td></td>
+          </tr>
+          <tr style="background: rgba(0,0,0,0.05); border-top: 1px solid rgba(0,0,0,0.05);">
+            <th style="background: rgba(255, 255, 255, 1); border-top: 1px solid rgba(255,255,255,1)"><strong></strong></th>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td><a href="https://canonical.com/microstack/docs">MicroStack</a></td>
+            <td></td>
+          </tr>
+          <!-- Tools -->
+          <tr style="background: rgba(0,0,0,0.025); border-top: 1px solid rgba(0,0,0,0.025);">
+            <th style="background: rgba(255, 255, 255, 1);  border-right: 1px solid rgba(0,0,0,0.05);"><strong>Tools</strong></th>
+            <td></td>
+            <td><a href="https://snapcraft.io/docs/snapcraft">Snapcraft</a></td>
+            <td><a href="https://documentation.ubuntu.com/chisel/en/latest/">Chisel</a></td>
+            <td><a href="https://documentation.ubuntu.com/rockcraft/">Rockcraft</a></td>
+            <td><a href="https://juju.is/docs/sdk/charmcraft">Charmcraft</a></td>
+            <td></td>
+          </tr>
+          <tr style="background: rgba(0,0,0,0.025); border-top: 1px solid rgba(0,0,0,0.025);">
+            <th style="background: rgba(255, 255, 255, 1)"><strong></strong></th>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td><a href="https://documentation.ubuntu.com/launchpad/en/latest/">Launchpad</a></td>
```

Comment:
Sorry to be a pain, but looking at this now, I think Launchpad should be moved up to the Management row in the same column (between MAAS and Juju). Thank you!